src/Message.js

```diff
@@ -82,7 +82,7 @@ export default class Message extends React.Component {
       <View>
         {this.renderDay()}
         <View style={[styles[this.props.position].container, {
-          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? 2 : 10,
+          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? this.props.marginSameUser : this.props.marginDifferentUser,
```

Comment:
@jayshah123  This is how github works, you stated that the master branch of your fork is the source of the PR so every change in it will be included, you could use a feature branch to achieve what you want